bot/db/models.py

```diff
@@ -18,14 +14,12 @@ class User(Base):
     first_name: Mapped[str]
     last_name: Mapped[str | None]
     username: Mapped[str | None]
-    is_vpn_enabled: Mapped[bool] = mapped_column(
-        default=True,  # по умолчанию True для новых пользователей
+    is_vpn_enabled: Mapped[bool | None] = mapped_column(
```

Comment:
```suggestion
    is_3x_ui_key_enabled: Mapped[bool | None] = mapped_column(
```

Давай переименуем поле в БД. Чтобы было понятнее.

Reply:
ok